### ISSUE TITLE
registry fails to compile with latest 9.6.6 LTS

### ISSUE THREAD

**Issue by tonicebrian** [etorreborre/registry#11]
When using `lts-22.43` in order to use GHC 9.6.6 I get this error:

```sh
       In the dependencies for registry-0.6.1.0:
         * resourcet must match >=1.1 && <1.3, but resourcet-1.3.0 is in the Stack configuration (latest matching version is 1.2.6).
         * semigroupoids must match >=5.0 && <5.4, but semigroupoids-6.0.1 is in the Stack configuration (latest matching version is 5.3.7).
```

Latest LTS has suparsed the upper bounds of registry on some libraries. Are they really breaking or could you just bump them to use latest `resourcet`  and `semigroupoids` ?

**Comment by tonicebrian** [comment]
I tried to bump libraries myself and the library compiles but the tests fail so it seems like a breaking change although I don't know about the severity. Let me know if I can be of any help here since we plan to use registry extensively.

**Comment by etorreborre** [comment]
@tonicebrian LTS-23 should hopefully provide `registry` soon: https://github.com/commercialhaskell/lts-haskell/pull/390.

**Comment by tonicebrian** [comment]
Thanks!! Looking forward to it

**Comment by tonicebrian** [comment]
Hi @etorreborre LTS 23 was released 2 days ago but registry is not part of it. Do you know the cause?

**Comment by etorreborre** [comment]
No idea. [The build file is still including it](https://github.com/commercialhaskell/lts-haskell/blob/master/build-constraints/lts-23-build-constraints.yaml#L4694).

**Comment by etorreborre** [comment]
@tonicebrian I have updated the dependencies that were not compatible with LTS 23.11. I hope it will be included with 23.12 now, and maybe a nightly snapshot in the meantime.

**Comment by etorreborre** [comment]
Hi @tonicebrian [everything looks good now](https://www.stackage.org/lts-23.14/package/registry-0.6.3.1).
